### DISCUSSION TITLE
Warning about serverExternalPackages when using with Next.js Turbopack

### DISCUSSION THREAD

**Discussion by dmorda** [resend/react-email#1816] [General]
I'm not sure if this is an issue with react-email or something specific to Turbopack. I recently upgraded several dependencies and upgraded to Next.js 15.1. Since then, I see the following error when each of my pages compiles in Next.js.

```
 ⚠ ./node_modules/@react-email/render/dist/node
Package prettier can't be external
The request prettier/plugins/html matches serverExternalPackages (or the default list).
The package resolves to a different version when requested from the project directory (3.4.2) compared to the package requested from the importing module (3.3.3).
Make sure to install the same version of the package in both locations.
```

I have a single package.json file in my setup and it have prettier 3.4.2 installed. If I install 3.3.3, that warning goes away.

Has anyone else encountered this issue? Maybe I have something configured wrong?

**Comment by samuelsogbesan** [comment]
I am running into the same error at the moment.

**Comment by davidchalifoux** [comment]
Installing prettier on my project resolved the error for me, but I shouldn't have to.
I believe it's because it's listed as a dependency here: https://github.com/resend/react-email/blob/canary/packages/render/package.json
Moving it to a dev depencency might prevent the warning.

**Reply by nyxz** [reply]
The render package uses prettier for it's core functions so it should not be moved to dev dependencies.

**Comment by coder2000** [comment]
I'm getting this exact error too

**Comment by CHC383** [comment]
Should be addressed in #2026
